data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg

```diff
@@ -230,7 +230,7 @@
 
         [message]
             role=Supporter
-            message= _ "Grim gods of darkness! The whole place is swarming with undead! They have raised corpses to do their work."
+            message= _ "Grim gods of darkness! The whole place is swarming with undead! Look, they’re mining!"
```

Comment:
The original is about horror and evil, setting up the enemy as bad, because necromancers have enslaved the dead. The new one misses that, and so I think the old one is better.

Reply:
Who are they, though? If you can find a wording that has that adds the horror but makes it clear I'll add it.

Reply:
I read it as an unspecified "they", meaning "whoever is running this place, even if we haven't seen them yet".

Reply:
Couldn't they also assume that the undead have risen themselves?

Reply:
FIxed it.